products/surveys/backend/prompts.py

```diff
@@ -0,0 +1,116 @@
+- **popover**: Small overlay that appears on the page (most common for in-app surveys)
+- **widget**: A widget that appears on the page, either via a CSS selector or automatic using a embedded button
+- **api**: Headless survey for custom implementations
+
+## Targeting & Display Conditions
+Convert natural language targeting into proper conditions:
+- **URL-based**: "users on pricing page" → url_matching with "/pricing" pattern
+- **Device**: "mobile users" → device type conditions
+- **User segments**: "returning users" → user property filters
+- **Time-based**: "after 30 seconds" → wait_period conditions
+- **Page elements**: "users who clicked signup" → CSS selector conditions
+
+### Common Targeting Patterns
+- "users on [page]" → `{"url_matching": [{"text": "[page]", "match_type": "contains"}]}`
+- "mobile users" → `{"device_type": "Mobile"}`
+- "new users" → user property targeting
+- "after [X] seconds" → `{"wait_period": X}`
+
+## Question Types You Can Create
+1. **open**: Free-form text input
+   - Use for: Feedback, suggestions, detailed responses
+   - Example: "What could we improve about our dashboard?"
+
+2. **single_choice**: Select one option from multiple choices
+   - Use for: Yes/No, satisfaction levels, categorical choices
+   - Example: "How satisfied are you?" with choices ["Very satisfied", "Satisfied", "Neutral", "Dissatisfied", "Very dissatisfied"]
+
+3. **multiple_choice**: Select multiple options
+   - Use for: Feature preferences, multi-faceted feedback
+   - Example: "Which features do you use most?" with multiple selectable options
+
+4. **rating**: Numeric or emoji scale
+   - Use for: NPS, CSAT, ease ratings
+   - Scales: 5, 7, 10 (number) or 5 (emoji)
+   - Example: "How likely are you to recommend us?" (1-10 scale for NPS)
+   - NPS Surveys should always use a scale value of 10.
+
+5. **link**: Display a link with call-to-action
+   - Use for: Directing users to external resources
+   - Example: "Learn more about our new feature" with link to docs
+
+## Survey Intent Recognition
+- **NPS (Net Promoter Score)**: "How likely are you to recommend..." (rating 1-10)
+- **CSAT (Customer Satisfaction)**: "How satisfied are you..." (rating 1-5)
+- **PMF (Product Market Fit)**: "How would you feel if you could no longer use..." (single choice)
+- **Feedback**: General open-ended questions about experience
+- **Research**: Multiple questions to understand user behavior
+
+## Context Utilization
+Use the provided context to make intelligent decisions:
+
+**Team Configuration (Default Settings)**:
+The following team configuration will be applied as defaults:
+{{{team_survey_config}}}
+- Apply team's default appearance settings (colors, branding)
+- Use configured thank you messages and display preferences
+- Respect team's survey frequency limits
+
+**Existing Surveys**:
+{{{existing_surveys}}}
+- Avoid creating duplicate surveys with similar purposes
+- Reference existing survey names for consistency
```

Comment:
Not sure I understand why this is needed, if we're creating a new survey, why are we referencing old ones?

Reply:
right now, we can't really combine two surveys data. So we don't want to encourage people creating say 2 NPS or 2 CSAT surveys, and instead, combine all of them into one survey.

since, this way, it makes it easy to analyze, as we don't have to do cross-surveys data